modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,149 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have client credentials of applications residing outside of Anypoint Platform if you were previously using a different API management. You must first migrate those client applications, with their client ID and client secret credentials, before you can start using Anypoint Platform.
+
+Because client IDs are unique in Anypoint Platform, you cannot reuse the same client ID in two different accounts. Your client applications can consume APIs whose organization is registered with a Anypoint Platform client provider or an external client provider. The instructions for these two scenarios differ:
+
+* <<migrate_with_internal,Migrate client applications for APIs with Anypoint Platform client providers>>
+* <<migrate_with_external,Migrate client applications for APIs with external client Providers>>
+
+== Prerequisites
+
+Before migrating the client application, ensure that you:
+
+* Obtain the required token and credentials:
+** Client ID and client secret for the client application to migrate
+** Anypoint Platform https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[token]
+* Review the <<considerations,scope and limitations>> to understand the limitations and scope of the migration task
+
+[[considerations]]
+== Scope and Limitations
+
+Before you migrate your client applications to Anypoint Platform, review the following scope and limitations: 
+
+* If the organization of the API you consume has registered an external client provider, for example, PingFederate or Okta, review the section: <<external_client,Organizations with Registered External Client Providers>>.
```

Comment:
"you consume"? You mean "your application"?
```suggestion
* If the organization of the API your application consumes has registered an external client provider (for example, PingFederate or Okta), review the section <<external_client,Organizations with Registered External Client Providers>>.
```

Reply:
Done! Thanks!